modules/eks-managed-node-group/variables.tf

```diff
@@ -408,6 +408,12 @@ variable "security_group_tags" {
   default     = {}
 }
 
+variable "launch_template_tag_specifications_tags" {
```

Comment:
```suggestion
variable "launch_template_tags" {
```

Reply:
you'll need to update in both places and fix the static checks (i.e. - `pre-commit run -a`)